src/civetweb.c

```diff
@@ -10843,8 +10843,9 @@ static int
 skip_to_end_of_word_and_terminate(char **ppw, int eol)
 {
 	/* Forward until a space is found - use isgraph here */
+	/* Extended ASCII characters are also treated as word characters. */
 	/* See http://www.cplusplus.com/reference/cctype/ */
-	while (isgraph((unsigned char)**ppw)) {
+	while ((unsigned char)**ppw > 127 || isgraph((unsigned char)**ppw)) {
```

Comment:
Fix `isgraph()` not working reliably for `(unsigned char)**ppw > 127`. This code was inspired by 

https://github.com/civetweb/civetweb/blob/073d636f8ad1a3dbda6026d3ddb82cfb764898bc/src/civetweb.c#L8116-L8117